src/main/java/io/frictionlessdata/tableschema/Table.java

```diff
@@ -278,20 +277,20 @@ public BeanIterator iterator(Class<?> beanType, boolean relations) throws Except
     }
     public String asJson() {
         try {
-            JSONArray arr = new JSONArray();
+            List<Map<String, Object>> arr = new ArrayList<>();
             List<Object[]> records = read();
             Schema schema = (null != this.schema) ? this.schema : this.inferSchema();
             for (Object[] rec : records) {
-                JSONObject obj = new JSONObject();
+                Map<String, Object> obj = new HashMap<>();
                 int i = 0;
                 for (Field field : schema.getFields()) {
                     Object s = rec[i];
                     obj.put(field.getName(), field.formatValueForJson(s));
                     i++;
                 }
-                arr.put(obj);
+                arr.add(obj);
             }
-            return arr.toString();
+            return JsonUtil.getInstance().serialize(arr);
```

Comment:
Couldn't this `asJson` method be simplified, by annotating the class appropriately for Jackson to serialize it directly?